test/checkpoint/test_checkpoint.c

```diff
@@ -336,22 +333,14 @@ static int
 usage(void)
 {
     fprintf(stderr,
-      "usage: %s "
-      "[-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t"
-      "[-l log] [-n ops] [-r runs] [-T table-config] [-t f|r|v]\n\t"
-      "[-W workers]\n",
+      "usage: %s [-C wiredtiger-config] [-c checkpoint] [-h home] [-k keys]\n\t[-l log] [-n ops] "
+      "[-r runs] [-T table-config] [-t f|r|v]\n\t[-W workers]\n",
       progname);
     fprintf(stderr, "%s",
-      "\t-C specify wiredtiger_open configuration arguments\n"
-      "\t-c checkpoint name to used named checkpoints\n"
-      "\t-h set a database home directory\n"
-      "\t-k set number of keys to load\n"
-      "\t-l specify a log file\n"
-      "\t-n set number of operations each thread does\n"
-      "\t-r set number of runs (0 for continuous)\n"
-      "\t-T specify a table configuration\n"
-      "\t-t set a file type ( col | mix | row | lsm )\n"
-      "\t-W set number of worker threads\n"
-      "\t-x use timestamps\n");
+      "\t-C specify wiredtiger_open configuration arguments\n\t-c checkpoint name to used named "
```

Comment:
Can we make an exception here? There's an obvious benefit to the previously used formatting.

Reply:
270093c